src/pages/aclmanagement.html

```diff
@@ -14,79 +14,162 @@ <h1 id="acl-management-view-title">
+                        <td>{{$index}}</td>
+                        <td class="reorder-cell">
+                            <div ng-if="editedRuleIndex === undefined" class="reorder-actions-group">
+                                <button ng-click="moveUp($index)" ng-if="$index > 0" class="btn btn-link move-up-btn"
+                                        gdb-tooltip="{{'acl_management.rulestable.actions.move_up' | translate}}">
+                                    <em class="fa fa-caret-up"></em>
+                                </button>
+                                <button ng-click="moveDown($index)" ng-if="$index < rulesModel.size() - 1"
+                                        class="btn btn-link move-down-btn"
+                                        gdb-tooltip="{{'acl_management.rulestable.actions.move_down' | translate}}">
+                                    <em class="fa fa-caret-down"></em>
+                                </button>
+                            </div>
+                        </td>
+                        <td class="subject-cell data">{{rule.subject}}</td>
+                        <td class="predicate-cell data">{{rule.predicate}}</td>
+                        <td class="object-cell data">{{rule.object}}</td>
+                        <td class="context-cell data">{{rule.context}}</td>
+                        <td class="role-cell data">{{rule.role}}</td>
+                        <td class="policy-cell data">{{rule.policy}}</td>
+                        <td class="actions-cell">
+                            <div ng-if="editedRuleIndex === undefined" class="crud-actions-group">
+                                <button ng-click="deleteRule($index)" class="btn btn-link delete-rule-btn"
+                                        gdb-tooltip="{{'acl_management.rulestable.actions.delete_rule' | translate}}">
+                                    <em class="icon-trash"></em>
+                                </button>
+                                <button ng-click="editRule($index)" class="btn btn-link edit-rule-btn"
+                                        gdb-tooltip="{{'acl_management.rulestable.actions.edit_rule' | translate}}">
+                                    <em class="icon-edit"></em>
+                                </button>
+                                <button ng-click="addRule($index + 1)" class="btn btn-link add-rule-btn"
+                                        gdb-tooltip="{{'acl_management.rulestable.actions.add_rule' | translate}}">
+                                    <em class="icon-plus"></em>
+                                </button>
+                            </div>
+                        </td>
+                    </tr>
+                    <tr ng-if="$index === editedRuleIndex" class="acl-rule edit-rule-row"
+                        ng-class="{'selected table-info': $index === editedRuleIndex}">
```

Comment:
`$index === editedRuleIndex` always be true

Reply:
I don't get this. We should discuss it. I'll eventually fix it with the next MR if needed.

Reply:
Sorry about confusing you `ng-class="{'selected table-info': $index === editedRuleIndex}"` is always true because tr tag has `ng-if="$index === editedRuleIndex"`.

Reply:
ahh I now see what you mean. Will update it with the next MR